BaseControl.Main/Program.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+
+namespace BaseControl.Main
+{
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+            List<string> splittedList = new List<string>();
+            List<int> amountOfPeople = new List<int>();
+            List<string> cities = new List<string>();
+            string pattern = @"";
+
+            Console.WriteLine("Enter list of the cities with population splitted by comma. \n\tExample: Kharkiv=1431000,Kiev=2804000,Las Vegas=603400");
+            Console.Write("\n\tInput a string: ");
+            //string inputString = "Kharkiv=1431000,Kiev=2804000,Las Vegas=603400";
+            string inputString = Console.ReadLine();
+
+            //if (string.IsNullOrEmpty(pattern))
+            //{
+            //    return;
+            //}
+
+            //try
+            //{
+            //    Regex.Match(inputString, pattern);
+            //}
+            //catch (ArgumentException)
+            //{
+            //    return;
+            //}
+
+            splittedList = inputString.Split(',').ToList();
+
+            foreach (var text in splittedList)
```

Comment:
It's better to use 'string' instead of 'var' because the type of the variable is not obvious. (Of course here as it is named 'text' it's rather obvious, but in general, in 'foreach' constructions the type should be used instead of 'var')